File: app/src/main/java/com/example/appconejitos/ReglasActivity.java
package com.example.appconejitos;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class ReglasActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_reglas);

        Button btnHome = findViewById(R.id.btnHome);
        btnHome.setOnClickListener(v -> {
            Intent intent = new Intent(this, InicioActivity.class);
            intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
            startActivity(intent);
            finish();
        });
    }
}

File: app/src/main/java/com/example/appconejitos/ReglasActivity.kt
package com.example.appconejitos

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class ReglasActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_reglas)

        findViewById<Button>(R.id.btnHome).setOnClickListener {
            finish()
        }
    }
}
